### PR TITLE
Lesson 1

### LINKED ISSUE #1: [DOC] - Lesson 1: Asset Management
## ℹ️  INFORMATION
Taking notes about asset management in Unity `<MORE_DATA_HERE>`

## 🚧  TASKS
- [ ] Create `lesson 1` directory
- [ ] Write notes using high-quality **markdown** syntax
- [ ] Follow GitFlow practise

## 🔗  REFERENCES
* [GitFlow](https://www.youtube.com/watch?v=G_iTZtnlf_U)
* [Unity](https://docs.unity3d.com/Manual/index.html)

## 📷  SCREENSHOTS
if applicable

### PR DESCRIPTION
**Pull request by whiteyhat**
## ℹ️  INFORMATION
This is a pull request to merge the lesson 1 from the Udemy course into the develop branch. This pr closes #1 

## 👞  STEPS
1. Verify the syntax quality of markdown
2. Check the GitFlow practice

## 🔗  REFERENCES
* Udemy 
* GitFlow

### CODE REVIEW

**Review by GreenyHat — APPROVED** [review]

**Review by GreenyHat — COMMENTED** [review]
.

File: LESSON 1/Asset Management.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+
+## Two main topics
+
+Tips         |  Theory
+------------ |  ------------
+Tips         |     Theory
+fsfdfgs      |     dfvsfgsgf
+dgeg         |   ddd
+
+solo hay una cosa importante en unity, como con todo, y es enfrentarse a la frustración, la curva de aprendizaje siempre será cuesta arriba, si no lo haces te esperan otros buenos puestos pero menos emocionantes, lo intentaré esxplicar con este diagrama:
```
Comment (whiteyhat) [inline LESSON 1/Asset Management.md:43]: Need the use of English here

File: LESSON 1/Asset Management.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+     id="path2"
+     style="fill:#110b09;fill-rule:nonzero" />
+</svg>
```
Comment (whiteyhat) [inline LESSON 1/Asset Management.md:31]: What is this?

**Review by whiteyhat — COMMENTED** [review]
It requires changes. To upload a flowchart just `drag & drop` in a GitHub issue and then `copy & paste` the URL in markdown format

File: LESSON 1/Asset Management.md (hunk 2 above)
Comment (GreenyHat) [reply LESSON 1/Asset Management.md:31]: it's an image

**Review by GreenyHat — COMMENTED** [review]
I don't know how introducing the diagram... I'm using MK syntax 🤔

**Comment by whiteyhat** [comment]
I will record a video to let you know @GreenyHat

File: LESSON 1/Asset Management.md (hunk 2 above)
Comment (whiteyhat) [reply LESSON 1/Asset Management.md]: Must be referenced from `image` tag as markdown tag

**Pull request closed without merging**